src/lsm/lsm_cursor.c

```diff
@@ -1730,6 +1730,11 @@ __wt_clsm_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner, cons
     WT_RET(__wt_config_gets_def(session, cfg, "bulk", 0, &cval));
     bulk = cval.val != 0;
 
+    if (bulk) {
```

Comment:
Just a style suggestion. Feel free to ignore. if (bulk && F_ISSET(session->txn, WT_TXN_RUNNING)) ?

Reply:
That's an artifact of me originally wanting to put breakpoints in the code easily, but it could now be changed as you suggest.

Reply:
I've just updated the code to combine into one if.